extension/content.js

```diff
@@ -22,6 +22,18 @@
     const START_ANIMATION = browser.extension.getURL("Start.json");
     const ERROR_ANIMATION = browser.extension.getURL("Error.json");
 
+    // https://gist.github.com/Rob--W/ec23b9d6db9e56b7e4563f1544e0d546
+    // slightly modified per https://www.owasp.org/index.php/XSS_(Cross_Site_Scripting)_Prevention_Cheat_Sheet#RULE_.231_-_HTML_Escape_Before_Inserting_Untrusted_Data_into_HTML_Element_Content
+    const escapeHTML = (str) => {
+      // Note: string cast using String; may throw if `str` is non-serializable, e.g. a Symbol.
+      // Most often this is not the case though.
+      return String(str)
+          .replace(/&/g, '&amp;')
+          .replace(/"/g, '&quot;').replace(/'/g, '&#x27;')
+          .replace(/</g, '&lt;').replace(/>/g, '&gt;')
+          .replace(/\//g, '&#x2F;');
```

Comment:
This slash is not necessary. For more background information, see https://security.stackexchange.com/questions/49852/why-should-xss-filters-escape-forward-slash